judgels-frontends/raphael/src/components/MathjaxWrapper/MathjaxWrapper.tsx

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
+import { HtmlText } from '../HtmlText/HtmlText';
 
 declare global {
   interface Window {
     MathJax: any;
   }
 }
 
-export class MathjaxWrapper extends React.Component {
+interface MathjaxProps {
+  text: string;
```

Comment:
Please make this `children` instead of a regular props, to make it similar to `HtmlText`

Reply:
Changed.